tlscanary/template/js/report_page.js

```diff
@@ -1,80 +1,78 @@
-function makeHeaderText(meta)
-{
-  var desc = "Fx " + meta.test_metadata.app_version + " " + meta.test_metadata.branch 
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+function makeHeaderText(meta) {
+  const desc = "Fx " + meta.test_metadata.app_version + " " + meta.test_metadata.branch 
           + " vs Fx " + meta.base_metadata.app_version + " " + meta.base_metadata.branch;
-  var time = meta.run_start_time.split(".")[0].replace("T","-").replace(":","-").replace(":","-");
+  const time = meta.run_start_time.split(".")[0].replace("T","-").replace(":","-").replace(":","-");
   window.document.getElementById("header").innerHTML = "<h3 class=\"header\">" + desc + "<br>" + time + "</h3>";
   window.document.title = "TLS Canary Report: " + desc;
 }
 
-function makeChartTab(uriList, fieldName)
-{
+function makeChartTab(uriList, fieldName) {
   makeFieldControl(fieldName);
   resizeChartCanvas();
-  var data = getPieChartData(uriList, fieldName);
+  const data = getPieChartData(uriList, fieldName);
   drawChart(data, fieldName);
   updateChartCaption(data, fieldName);
 
-  var div = window.document.getElementById("chart_text");
+  const div = window.document.getElementById("chart_text");
   div.style.position = "absolute";
   div.style.top = "0";
   div.style.left = $("#chart_canvas").width() * 1.2 + "px";
 }
 
-function makeFieldControl(fieldName)
-{
+function makeFieldControl(fieldName) {
   var html = "";
   html += "<h3>Field:&nbsp;&nbsp;<select id=\"fieldNames\" name=\"fieldNames\" >";
   var columns = getVisibleColumns();
-  for (var i=0;i<columns.length;i++)
+  for (var i = 0;i < columns.length;i++)
```

Comment:
Almost there. According to MDN style guide, for loop formatting is

```js
for (...; ...; ...) {
    ...
}
```

Please also check for loops elsewhere.